pages/docs/adapters/calculix/adapter-calculix-configure.md

```diff
@@ -23,30 +23,37 @@ precice-config-file: ../precice-config.xml
 ```
 
 The adapter allows to use several participants in one simulation (e.g. several instances of Calculix if several solid objects are taken into account). The name of the participant "Calculix" must match the specification of the participant on the command line when running the executable of "CCX" with the adapter being used (this is described later). Also, the name must be the same as the one used in the preCICE configuration file *precice-config.xml*.  
-One participant may have several FSI interfaces. Note that each interface specification starts with a dash.  
-For FSI simulations the mesh type of an interface is always "nodes-mesh", i.e. the mesh is defined node-wise, not element-wise. The name of this mesh, "Calculix_Mesh", must match the mesh name given in the preCICE configuration file.  
+One participant may have several coupling interfaces. Note that each interface specification starts with a dash.
+Depending on the data you need to read and write, you can use a mesh made of face centers ("faces-mesh" or just "mesh") or a mesh made of actual CalculiX nodes.
+For FSI simulations the mesh type of an interface is always "nodes-mesh", as forces and displacement are defined on nodes. The name of this mesh, "Calculix_Mesh", must match the mesh name given in the preCICE configuration file. In CHT simulations, faces mesh are usually chosen, as they are needed to apply heat fluxes or convective heat transfer.
```

Comment:
Consistency: 
```suggestion
For FSI simulations the mesh type of an interface is always "nodes-mesh", as forces and displacements are defined on nodes. The name of this mesh, "Calculix_Mesh", must match the mesh name given in the preCICE configuration file. In CHT simulations, "faces-mesh" are usually chosen, as they are needed to apply heat fluxes or convective heat transfer.
```

(even though I would prefer `node-based meshes` and `face-based meshes`)

Reply:
The convention seems to favor keywords instead of natural language.